cli/perimeterCommands.js

```diff
@@ -0,0 +1,170 @@
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+const config = require('./configCommands.js');
+const prompts = require('prompts');
+const fs = require('fs').promises;
+
+const perimeterCommands = {
+
+    async processPerimeterCommand(args) {
+        let action = args[0];
+        if (!action) {
+            action = (
+                await prompts({
+                    type: 'select',
+                    name: 'value',
+                    message: 'Perimeter action',
+                    choices: [
+                        { title: 'Create', value: 'create' },
+                        { title: 'Delete', value: 'delete' },
+                    ],
+                })
+            ).value;
+            if (!action) {
+                console.log('Perimeter action is required');
+                return;
+            }
+        }
+        switch (action) {
+            case 'create':
+                await this.createPerimeter(args.slice(1));
+                break;
+            case 'delete':
+                await this.deletePerimeter(args.slice(1));
+                break;
+            default:
+                console.log(`Unknown perimeter action : ${action}
+                `);
+                await this.printHelp();
+                break;
+        }
+    },
+
+
+    async createPerimeter(args) {
+        let perimeterFile = args[0];
+
+        if (!perimeterFile) {
+            perimeterFile = (
+                await prompts({
+                    type: 'text',
+                    name: 'value',
+                    message: 'JSON perimeter file name '
+                })
+            ).value;
+            if (!perimeterFile) {
+                console.log('JSON perimeter file name is required');
+                return;
+            }
+        }
+
+        let fileContent;
+        try {
+            fileContent = await fs.readFile(perimeterFile, 'utf8');
+        }
+        catch (error) {
+            console.error('Failed to read perimeter file', perimeterFile);
+            console.error('Error:', error);
+            return;
+        }
+        const url = `${config.getConfig('url')}:2002/users/perimeters`;
+        const token = config.getConfig('access_token');
+        const options = {
+            method: 'POST',
+            body: fileContent,
+            headers: {
+                'Content-Type': 'application/json',
+                'Authorization': `Bearer ${token}`
+            }
+        };
+
+        let response;
+        try {
+            response = await fetch(url, options);
+        } catch (error) {
+            console.error('Failed to create perimeter');
+            console.error('Error:', error);
+            return;
+        }
+
+        if (response.ok) {
+            console.error('Perimeter created successfully');
+            const result = await response.json();
+            console.error(result);
```

Comment:
no need to print the result in this case (as we will use it later for loading all perimeters in loadTestConf , it will print to much log on the terminal)

Reply:
Done.